genservice.sh

```diff
@@ -1,4 +1,4 @@
-#!bin/bash
+#!/bin/bash
```

Comment:
legitimate question - what does this line actually do? #bashnoob

Reply:
Its for when you run the command lets compiler know what to run with the file
Could do #!/path/to/node
in app.js
Then you could run ./app.js